daemon/src/db.rs

```diff
@@ -160,6 +173,14 @@ impl Connection {
         .bind(&cfd.settlement_time_interval_hours().whole_hours())
         .bind(&cfd.quantity())
         .bind(&cfd.counterparty_network_identity())
+        .bind(&cfd.counterparty_libp2p_identity().unwrap_or({
+            tracing::debug!(
+                order_id=%cfd.id(),
+                taker_id=%cfd.counterparty_network_identity(),
+                "Inserting deprecated CFD with placeholder peer-id!"
+            );
```

Comment:
❓ Is this expected to happen only once per "old" CFD?

Reply:
This is only expected to happen if a taker on a previous version opens a Cfd with a maker on the new version.
Unfortunately we cannot really distinguish because taker and maker share the same db and model.